rsdockerspawner/rsdockerspawner.py

```diff
@@ -527,8 +528,6 @@ def __slot_alloc(self, no_raise=False):
                 yield self.__init_class()
         pool, s = self.__slot_for_container(n)
         if s:
-            # Ensures pool_gc doesn't select this slot
-            s.activity_secs = time.time()
```

Comment:
I think this still needs to be here. This is the case where the slot is allocated, and we don't want it to be reallocated so we set activity_secs to the most recent time. Note the `yield` above. This could be a long time.

Reply:
But, if we do this then I think long running containers will all have the same activity_secs so it is random which one is garbage collected.

Reply:
time.time is only used when the slot is allocated, which will only happen when it is started. The reason for setting activity_secs here is that there are yields above so you don't know how long time has passed. Once allocated, we want to make sure that the time gets updated so it isn't gc'd.

Reply:
We discussed that it is correct to remove this line. The activity secs should not be left as is so that setting the activity_secs to the previous slots activity_secs persists.